### DISCUSSION TITLE
Define binding which applies to specified window other than the one currently focused

### DISCUSSION THREAD

**Discussion by asterixpound** [i3/i3#5852] [Q&A] [answered]
I am interested in defining a binding which brings a specific window from a workspace other than the one currently focused to the current workspace. In particular, I would like to use the same criteria  as used for `for_window` such as `class`, `title`, etc.

Is this possible?

**Comment by okraits** [comment, marked as answer]
Yes, you can combine command criteria with the command to move a container to a specific workspace:

* https://i3wm.org/docs/userguide.html#command_criteria
* https://i3wm.org/docs/userguide.html#_changing_named_workspaces_moving_to_workspaces

**Reply by asterixpound** [reply]
Thanks, this is exactly what I wanted. Now I am using the following binding in my config:

```
bindsym $mod+n [title="^Window_Title$"] move container to workspace current, focus
```
